builtin/logical/pki/path_root.go

```diff
@@ -243,14 +247,23 @@ func (b *backend) pathCASignIntermediate(
 	case "pem":
 		resp.Data["certificate"] = cb.Certificate
 		resp.Data["issuing_ca"] = cb.IssuingCA
+		resp.Data["issuing_ca_chain"] = cb.IssuingCAChain
 
 	case "pem_bundle":
-		resp.Data["certificate"] = fmt.Sprintf("%s\n%s", cb.Certificate, cb.IssuingCA)
+		resp.Data["certificate"] = cb.ToPEMBundle()
```

Comment:
I added the ToPemBundle helper later in the process and didn't find all the appropriate case.  In the prior case dealing with CSRs, I thought we needed to output different things but it looks like that's not the case since there is no private key available when using a CSR.